bblfsh/client.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+
+import grpc
+
+# The following two insertions fix the broken pb import paths
+sys.path.insert(0, os.path.join(os.path.dirname(__file__),
+                                "github/com/bblfsh/sdk/protocol"))
+sys.path.insert(0, os.path.dirname(__file__))
+from bblfsh.github.com.bblfsh.sdk.protocol.generated_pb2 import ParseUASTRequest
+from bblfsh.github.com.bblfsh.sdk.protocol.generated_pb2_grpc import ProtocolServiceStub
+
+
+class BblfshClient(object):
+    """
+    Babelfish gRPC client. Currently it is only capable of fetching UASTs.
+    """
+
+    def __init__(self, endpoint):
+        """
+        Initializes a new instance of BblfshClient.
+
+        :param endpoint: The address of the Babelfish server, \
+                         for example "0.0.0.0:9432"
+        :type endpoint: str
+        """
+        self._channel = grpc.insecure_channel(endpoint)
+        self._stub = ProtocolServiceStub(self._channel)
+
+    def fetch_uast(self, file_path, language=None, contents=None):
```

Comment:
Please, call this `parse_uast` so that it matches the protocol as much as possible.